notebooks/eight_schools.ipynb

```diff
@@ -0,0 +1,533 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Eight Schools - A Stan/Edward Comparison\n",
```

Comment:
Since this notebook is in Edward: maybe rename to "Edward/Stan", and move the Stan section below the Edward section?